src/screens/authentication.js

```
@@ -75,13 +76,13 @@ class AuthenticationScreen extends React.PureComponent<Props, State> {
 
   handleSignOut = () => this.props.navigation.popToTop();
 
-  handleSignIn = async (token?: string) => {
-    await this.props.signIn(token);
+  handleSignIn = async (authenticationType: ?AuthenticationType, token?: string) => {
```

should throw a flow error.